.github/workflows/ci.yml

```diff
@@ -10,9 +10,9 @@ on:
 
 env:
```

Comment:
i'm wondering if this CI workflow should also be updated to remove the publishing step since there would be an explicit publishing workflow to replace it - let me check with others to see if there's guidance on that too!

Reply:
Aye!

Reply:
Given the guidance in https://github.com/crossplane/org/issues/88#issuecomment-2714300280, you have the choice of publishing on every CI run or having publishing be a manual/conscious step with the new .github/workflows/publish-provider-package.yaml added in this PR. I think it's totally reasonable to remove the publishing from https://github.com/crossplane-contrib/provider-upjet-digitalocean/blob/main/.github/workflows/ci.yml#L241 and then only publish when you want to by running the new specific publishing action.